packages/plugin-dlx/sources/commands/dlx.ts

```diff
@@ -88,8 +88,8 @@ export default class DlxCommand extends BaseCommand {
         await xfs.writeFilePromise(targetYarnrc, `enableGlobalCache: true\nenableTelemetry: false\n`);
       }
 
-      const pkgs = typeof this.pkg !== `undefined`
-        ? [this.pkg]
+      const pkgs = typeof this.packages !== `undefined`
+        ? [...this.packages]
```

Comment:
Do you need the spread? `this.packages` should already be an array

Reply:
I don't no, missed that